### DISCUSSION TITLE
How to use dir, not supported or a Bug?

### DISCUSSION THREAD

**Discussion by CriTT90** [yandeu/five-server#86] [Help / Questions]
I am trying to navigate through the folder structure to open a specific file. 

```
$local_dir = $_SERVER["DOCUMENT_ROOT"].'/aWohnungen/';
if ( is_dir ( $local_dir ))
{
    // öffnen des Verzeichnisses
    if ( $handle = opendir($local_dir) )
    {
        // einlesen der Verzeichnisses
        while (($file = readdir($handle)) !== false)
        {
            echo "<li>Dateiname: ";
            echo $file;

            echo "<ul><li>Dateityp: ";
            echo filetype( $file );
            echo "</li></ul>\n";
        }
        closedir($handle);
    }
}
```

Page leaves Blank. If i chance $local_dir = "."; i get as Outcome this:

Dateiname: .
Dateityp: dir
Dateiname: ..
Dateityp: dir
Dateiname: bin
Dateityp: dir
Dateiname: chrome_100_percent.pak
Dateityp: file
Dateiname: chrome_200_percent.pak
Dateityp: file
Dateiname: Code.exe
Dateityp: file
Dateiname: Code.VisualElementsManifest.xml
Dateityp: file
Dateiname: d3dcompiler_47.dll
Dateityp: file
Dateiname: ffmpeg.dll
Dateityp: file
Dateiname: icudtl.dat
Dateityp: file
Dateiname: libEGL.dll
Dateityp: file
Dateiname: libGLESv2.dll
Dateityp: file
Dateiname: locales
Dateityp: dir
Dateiname: policies
Dateityp: dir
Dateiname: resources
Dateityp: dir
Dateiname: resources.pak
Dateityp: file
Dateiname: snapshot_blob.bin
Dateityp: file
Dateiname: tools
Dateityp: dir
Dateiname: unins000.dat
Dateityp: file
Dateiname: unins000.exe
Dateityp: file
Dateiname: unins000.msg
Dateityp: file
Dateiname: v8_context_snapshot.bin
Dateityp: file
Dateiname: vk_swiftshader.dll
Dateityp: file
Dateiname: vk_swiftshader_icd.json
Dateityp: file
Dateiname: vulkan-1.dll
Dateityp: file

This is not the content from root or anywhere else from the project.

**Comment by yandeu** [comment]
I see. Please have a look and read https://github.com/yandeu/five-server-with-php
